test/Microsoft.ML.TestFramework/DataPipe/TestDataPipe.cs

```diff
@@ -14,6 +14,7 @@
 using Microsoft.ML.Runtime.Model;
 using Microsoft.ML.Runtime.TextAnalytics;
 using Xunit;
+using Microsoft.ML.Transforms;
```

Comment:
>using [](start = 0, length = 5)

(placeholder for Tom's comment)

Reply:
Hey Ivan, Pete asked me to tell you to sort your usings. :)

---
In reply to: [218936325](https://github.com/dotnet/machinelearning/pull/944#discussion_r218936325) [](ancestors = 218936325)